modules/raft/src/main/java/org/apache/ignite/raft/jraft/disruptor/StripedDisruptor.java

```diff
@@ -121,7 +124,7 @@ public StripedDisruptor(
      * @param eventFactory Event factory for the Striped disruptor.
      * @param stripes Amount of stripes.
      * @param supportsBatches If {@code false}, this stripe will always pass {@code true} into
-     *      {@link EventHandler#onEvent(Object, long, boolean)}. Otherwise, the data will be provided with batches.
+     *         {@link EventHandler#onEvent(Object, long, boolean)}. Otherwise, the data will be provided with batches.
```

Comment:
Formatting seems incorrect

Reply:
The same as above. I have checked the results again.